### PR TITLE
Fixed minor bugs, renamed some files to be more descriptive

### PR DESCRIPTION
**Pull request by haseebakhan10**
In this PR I changed the customer view so it does not create a new table for each item, and entered the table to make it look nicer

Sorry about the commit name, I changed it

### CODE REVIEW

File: src/main/java/ShopifyProj/ThymeController.java [hunk 1]
```diff
@@ -41,26 +41,26 @@ private Shop getShopById(int aShopId) {
```
Comment (haseebakhan10) [inline src/main/java/ShopifyProj/ThymeController.java]: I renamed some HTML pages for consistency and to make it more obvious what the page will display

File: src/main/resources/templates/CreateShop.html [hunk 2]
```diff
@@ -19,14 +19,15 @@ <h3>Your Current Shops:</h3>
             <th>Tags</th>
         </tr>
         <tr th:each="shop: ${shops}">
-            <td th:text="${shop.id}"></td>
-            <td th:text="${shop.shopName}"></td>
-            <td th:text="${shop.tags}"></td>
-            <td>
+            <td style="text-align:center" th:text="${shop.id}"></td>
```
Comment (haseebakhan10) [inline src/main/resources/templates/CreateShop.html]: Centred tables throughout the application for consistency and to make it look more polished.

**Comment by haseebakhan10** [comment]
> The renaming and centering of the tables looks good.
> 
> The "Remove Item" button in MerchantShopView doesn't seem to be working. It's not deleting the item.

I ran it locally and it seemed to be working. I didn't change the logic so it should still be functional. Can you please try again and let me know if you still have the issue

**Comment by BramSrna** [comment]
> > The renaming and centering of the tables looks good.
> > The "Remove Item" button in MerchantShopView doesn't seem to be working. It's not deleting the item.
> 
> I ran it locally and it seemed to be working. I didn't change the logic so it should still be functional. Can you please try again and let me know if you still have the issue

I retried it and it's working. Sorry about that, I'm not sure what happened.

**Pull request merged**